src/index.tsx

```diff
@@ -230,3 +260,7 @@ const signInWithRedirect = (): void => {
       });
   }
 };
+
+const getCookieValue = (name: string) : string => (
```

Comment:
```suggestion
const getCookieValue = (name: string): string => (
```
wasn't formatted right